src/names.c

```diff
@@ -1145,6 +1154,80 @@ void insert_spec(nametab_t *tab, tree_t spec, spec_kind_t kind,
    *p = s;
 }
 
+void set_impl_label_proc_cnt(tree_t t, nametab_t *tab)
```

Comment:
This name is quite obscure: to me "impl" means "implementation". How about `continue_labelling_from()`?

Reply:
I used `continue_proc_labelling_from` to be explicit its counter of processes.